src/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -0,0 +1,79 @@
+import json from "./AllowedVocab.json"
+
+/**
+ * Checks, if all words used, are allowed.
+ * 
+ * 
+ * 
+ */
+
+function preprocessingText(text: string){
+    var cleantext = eliminatePunctuation(text);
+    var textarray = transformTextToStringArray(cleantext);
+    return textarray;
+}
+
+export function checkInput(text: string){
+    let preprocessedText : string[] = preprocessingText(text);
+    let issue : string[] = []; //collect all wrong words in a stringarray
```

Comment:
You can do this in an easier way:
`const invalidWords = preprocessedText.filter(word => !json.foo.includes(word));`